docs/parameters.md

```diff
@@ -0,0 +1,53 @@
+# Storage Class Parameters
```

Comment:
Should we add a section on PersistentVolume (PV)? We designed the volumeAttributes (dnsName, volumePath,) so it would be a good reference.

Reply:
Agreed, I'll add a section for this